src/pkg/cluster/state.go

```diff
@@ -89,6 +90,9 @@ func (c *Cluster) InitZarfState(ctx context.Context, initOptions types.ZarfInitO
 		}
 		// Mark existing namespaces as ignored for the zarf agent to prevent mutating resources we don't own.
 		for _, namespace := range namespaceList.Items {
+			if namespace.Name == "zarf" {
```

Comment:
Why is this being added?

Reply:
Apply doesn't delete existing labels, but adds to them. Whereas update fully replaces the object. Previously if the Zarf namespace already existed on init, we would add the `zarf-agent: ignore` to it, but it would get replaced in the update a few lines down. Now that we apply, the `zarf` ns would keep the `zarf-agent: ignore` label, so I excluded it from receiving the label